### ISSUE TITLE
FEATURE REQUEST: add defaultColumnProps prop

### ISSUE THREAD

**Issue by capaj** [icflorescu/mantine-datatable#340]
**Is your feature request related to a problem? Please describe.**
I find that often I want my cells to have this `cellsStyle`
```
             overflow: 'hidden',
              textOverflow: 'ellipsis',
              whiteSpace: 'nowrap'
```

**Describe the solution you'd like**
would be nice if I could override this globally for all my cells instead of having to define this for each column

**Describe alternatives you've considered**
I can define it for each column

**Additional context**

**Comment by icflorescu** [comment]
Definitely sounds like a useful feature, but I think I'd rather call it `defaultColumnProps`. What do you think?

**Comment by capaj** [comment]
yes, that would be better. That way you can override other column props as well :rocket:

**Comment by capaj** [comment]
I know I have previously asked for `defaultColumnRender`, but yeah, having `defaultColumnProps` sounds more useful. There I can override both render and cellsStyle

**Comment by icflorescu** [comment]
This should be available in a few minutes as soon as `v2.7.0` hits the npm registry.
